sovereignApplications/confidential/hrAppWorkload/docs/02-architecture.md

```diff
@@ -8,13 +8,13 @@ The application architecture implements the following Azure capabilities on a th
 
 1. **Sovereign Landing Zone** - [The Sovereign Landing Zone (SLZ)](https://github.com/Azure/sovereign-landing-zone/blob/main/docs/01-Overview.md) is the recommended framework to secure deployment of workloads.
 
-1. **Sensitive Data** - [Azure SQL DB - Always Encrypted with secure enclaves](https://docs.microsoft.com/azure/azure-sql/database/always-encrypted-with-secure-enclaves-landing). For a sovereign confidential `HR` database, the `SSN` and `Salary` columns are encrypted using the Column Master Key ([`CMK`](https://learn.microsoft.com/sql/relational-databases/security/encryption/configure-always-encrypted-keys-using-ssms?view=sql-server-ver16)). This capability is the data-tier in architecture diagram.
+1. **Sensitive Application Logic** - [Azure Confidential VM - with AMD SEV-SNP](https://learn.microsoft.com/azure/confidential-computing/virtual-machine-solutions-amd): Used for hosting an ASP.NET web app that queries the `HR` Azure SQL DB using the [ADO.NET driver](https://docs.microsoft.com/sql/connect/ado-net/microsoft-ado-net-sql-server?view=azuresqldb-current) and Azure [Confidential ledger client library for .NET](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/security.confidentialledger-readme?view=azure-dotnet) to persist Sensitive Logs generated on the web app (in this case, query history). This capability is the `FRONT TIER` in architecture diagram.
```

Comment:
Sometime we have `docs.microsoft` and sometimes we have `learn.microsoft`, should choose one domain to be consistent?

Reply:
If we make that change I don't think it is for this PR to deal with

Reply:
I see, will do another PR for having conistency across all articles. Thanks